### ISSUE TITLE
Tutorial Request: How to migrate Existing Drupal Site to Google Cloud.

### ISSUE THREAD

**Issue by Akash260219** [GoogleCloudPlatform/community#270]
Hi Guys ,

Please let me know the step by step process for migration of my Drupal site to google cloud.

**Comment by bshaffer** [comment]
Hello @Akash260219 ! What version of Drupal are you using? We have a sample of running Drupal8 on Google Cloud [here](https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/appengine/flexible/drupal8) which you may find helpful.

**Comment by Akash260219** [comment]
Hello @bshaffer ! I am using Drupal 7. Is the procedure same for Drupal 7 as of Drupal 8??

**Comment by Extra-lightwill** [comment]
Hi @bshaffer, I can't connect to the database following the walkthrough in the link above. 

I'm using the following details:

```
drush site-install \
  --locale=en \
  --db-url=mysql://root:GSC_DATABASE_INSTANCE_PASSWORD@localhost/DATABASE_INSTANCE_ID \
  --site-name='GSC_PROJECT_NAME' \
  --site-mail=GOOGLE_ACCOUNT_EMAIL \
  --account-name GOOGLE_ACCOUNT_NAME \
  --account-mail GOOGLE_ACCOUNT_EMAIL \
  --account-pass GOOGLE_ACCOUNT_PASSWORD
```

Are these correct?

**Comment by Extra-lightwill** [comment]
@bshaffer Please can you provide an example of how to execute the --db-url command with mock data and explain it in layman's terms?

**Comment by raquel-campuzano** [comment]
Hello @Akash260219, maybe this guide can help you. It explains how to create a backup of your Drupal existing data and then, how to copy and restore it to a fresh Google Cloud instance. Hope you find it useful. https://docs.bitnami.com/google/how-to/migrate-drupal/
